index.d.ts

```diff
@@ -171,6 +171,28 @@ declare namespace algoliasearchHelper {
       ) => void
     ): void;
 
+    /**
+     * Start the search for answers with the parameters set in the state.
+     * This method returns a promise.
+     * @param {Object} options - the options for answers API call
+     * @param {string[]} options.attributesForPrediction - Attributes to use for predictions. If empty, `searchableAttributes` is used instead.
+     * @param {string[]} options.queryLanguages - The languages in the query. Currently only supports ['en'].
+     * @param {number} options.nbHits - Maximum number of answers to retrieve from the Answers Engine. Cannot be greater than 1000.
+     */
+    findAnswers(options: {
+      attributesForPrediction: string[];
+      queryLanguages: string[];
+      nbHits: number;
+    }): Promise<{
+      hits: Array<any & {
```

Comment:
can you not use the return type of findAnswers on the client?

Reply:
this method likely should be generic, like the client one

Reply:
I've updated the dependency and now it's using the type from the client:
https://github.com/algolia/algoliasearch-helper-js/pull/804/commits/da8af1338fe8038ef29b3786be0a0c789d6a2daf